scripts/wireguard/makeCONF.sh

```diff
@@ -110,52 +110,58 @@ done
 NET_REDUCED="${pivpnNET::-2}"
 
```

Comment:
The unrelated changes you do here generally make sense, doing all writes to the config files in one batch instead of multiple small writes. But similarly to formatting enhancements, I'd suggest to do them in a dedicated PR as well, keeping this one minimal for the changes required to enable Alpine support only.